src/reclaim_python_sdk/utils/proof_utils.py

```
@@ -54,23 +54,26 @@ async def create_link_with_template_data(template_data: TemplateData) -> str:
+        async with httpx.AsyncClient() as client:
+            response = await client.get(DEFAULT_ATTESTORS_URL)
+            if response.status_code != 200:
+                response.read()
+                raise Exception(f"Failed to fetch witness addresses: {response.status_code}")
+            
+            res_data = response.json()
+            # The JS SDK expects data: { address: string }[] but handles res.data
+            address_list = res_data.get("data", [])
+            witnesses = [w["address"].lower() for w in address_list if "address" in w]
+            return witnesses
     except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)